tools/test.py

```diff
@@ -180,10 +180,12 @@ def main():
                pin_memory=False,
            )),
         **dict((k, cfg.data[k]) for k in [
+                   'samples_per_gpu',
```

Comment:
No need to add `samples_per_gpu` here, it will be added back in L195
https://github.com/open-mmlab/mmocr/blob/fba996d07c577d164b29acbd4b2b84870847a33d/tools/test.py#L195

Reply:
I cannot remember why it was special cased at L195...

Reply:
https://github.com/open-mmlab/mmocr/blob/fba996d07c577d164b29acbd4b2b84870847a33d/tools/test.py#L145-L164